app/services/election_service.rb

```diff
@@ -1,18 +1,18 @@
 module ElectionService
   def self.create_candidate(candidate, user)
     candidate.user = user
-    candidate.validate!
-    if !candidate.election.current_posts.exists?(candidate.post.id)
-      candidate.errors.add(:post, I18n.t('election.not_allowed_post'))
-      return false
-    end
 
-    candidate.save!
+    if candidate.save
+      ElectionMailer.candidate_email(candidate).deliver_now
+      true
+    else
+      false
+    end
   end
 
   def self.destroy_candidate(candidate)
     if candidate.editable?
-      candidate.destroy!
+      candidate.destroy
```

Comment:
https://github.com/fsek/web/blob/david-candidates/app/controllers/elections/candidates_controller.rb#L42

But I guess I could just as well do it with a bang.